src/main/java/com/google/devtools/build/lib/bazel/bzlmod/ModuleFileGlobals.java

```diff
@@ -432,31 +432,42 @@ public void registerToolchains(boolean devDependency, Sequence<?> toolchainLabel
             named = true,
             positional = false,
             defaultValue = "False"),
+        @Param(
+            name = "isolated",
+            doc =
+                "If true, this usage of the module extension will be isolated from all other "
```

Comment:
we'll need to explain a bit better what "isolation" means here. question -- why "in particular those in other modules"? It creates isolation just as much for other usages in the current module, right?

Reply:
I expanded the docs, please take another look. I would prefer to avoid the term "workspace" in favor of the less overloaded term "repository".

Reply:
I like the new docs!